src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_help.py

```diff
@@ -0,0 +1,530 @@
+    type: command
+    short-summary: Creates Authorization rule for the given Service Bus Queue
+    examples:
+        - name: Creates Authorization rules for Queue
+          text: az servicebus queue authorization-rule create --resource-group myresourcegroup --namespace-name mynamespace --queue-name myqueue --name myauthorule --rights Listen
+"""
+
+helps['servicebus queue authorization-rule update'] = """
+    type: command
+    short-summary: Updates Authorization rule for the given Service Bus Queue
+    examples:
+        - name: Updates Authorization rules for Queue
+          text: az servicebus queue authorization-rule update --resource-group myresourcegroup --namespace-name mynamespace --queue-name myqueue --name myauthorule --rights Send
+"""
+
+helps['servicebus queue authorization-rule show'] = """
+    type: command
+    short-summary: shows the details of Authorization Rule
+    examples:
+        - name: shows the details of Authorization Rule
+          text: az servicebus queue authorization-rule show --resource-group myresourcegroup --namespace-name mynamespace --queue-name myqueue --name myauthorule
+"""
+
+helps['servicebus queue authorization-rule list'] = """
+    type: command
+    short-summary: shows the list of Authorization Rule by Queue
+    examples:
+        - name: shows the list of Authorization Rule by Queue
+          text: az servicebus queue authorization-rule show --resource-group myresourcegroup --namespace-name mynamespace --queue-name myqueue
+"""
+
+helps['servicebus queue authorization-rule keys list'] = """
+    type: command
+    short-summary: Shows the connection strings of Authorization Rule for the Queue.
+    examples:
+        - name: Shows the connection strings of Authorization Rule for the queue.
+          text: az servicebus queue authorization-rule list-keys --resource-group myresourcegroup --namespace-name mynamespace --queue-name myqueue --name myauthorule
+"""
+
+helps['servicebus queue authorization-rule keys renew'] = """
+    type: command
+    short-summary: Regenerate the connection strings of Authorization Rule for the namespace.
+    examples:
+        - name: Regenerate the connection strings of Authorization Rule for the namespace.
+          text: az servicebus queue authorization-rule regenerate-keys --resource-group myresourcegroup --namespace-name mynamespace --queue-name myqueue --name myauthorule --key PrimaryKey
+"""
+
+helps['servicebus queue authorization-rule delete'] = """
+    type: command
+    short-summary: Deletes the Authorization Rule of the Queue.
+    examples:
+        - name: Deletes the Authorization Rule of the queue.
+          text: az servicebus queue authorization-rule delete --resource-group myresourcegroup --namespace-name mynamespace --queue-name myqueue --name myauthorule
+"""
+
+helps['servicebus topic create'] = """
+    type: command
+    short-summary: Creates the Service Bus Topic
+    examples:
+        - name: Create a new queue.
```

Comment:
Is this description correct?
Is a queue the same as a service bus topic?

Reply:
corrected the description.